### ISSUE TITLE
switch back to serverless 

### ISSUE THREAD

**Issue by vincentsarago** [RemotePixel/remotepixel-tiler#8]

**Comment by DanSchoppe** [comment]
Just curious, did you have a bad experience with Terraform? I've been using it for about a year now and like it so far 😄

**Comment by vincentsarago** [comment]
👋 @DanSchoppe 
so lately with the update to terraform 0.12 everything broke and I had hard time to recover. 

I switched to terraform to overcome some serverless issue with API Gateway binary response and thus I created https://github.com/developmentseed/tf-lambda-proxy-apigw to make everything easier. But then terraform update happened and also it's now simpler to set binary response in serverless https://github.com/developmentseed/cogeo-mosaic/blob/master/serverless.yml#L30-L32

I really tried to embrace Terraform but now it's giving me hard time.
